templates/etc_systemd_system_strong-pm.service.d_custom.conf.j2

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=/opt/node-{{ node_version }}-linux-x64/bin/node /usr/lib/node_modules/strong-pm/bin/sl-pm.js --listen 8701 --base /var/lib/strong-pm --base-port 3000 --driver direct {% if strong_pm_skip_default_install %} --skip-default-install {% endif %}}
```

Comment:
We can't hardcode the path like that - you want to have a step that register the `ExecStart` string and re-use it here; this is the only way to support various versions install path of `node`